src/main/java/de/retest/recheck/review/ignore/ColorDiffFilter.java

```diff
@@ -0,0 +1,125 @@
+package de.retest.recheck.review.ignore;
+
+import static java.lang.Float.parseFloat;
+import static java.lang.Math.abs;
+import static java.lang.Math.min;
+import static java.lang.Math.round;
+
+import java.awt.Color;
+import java.io.Serializable;
+import java.util.Optional;
+import java.util.regex.MatchResult;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import de.retest.recheck.ignore.Filter;
+import de.retest.recheck.review.ignore.io.RegexLoader;
+import de.retest.recheck.ui.descriptors.Element;
+import de.retest.recheck.ui.diff.AttributeDifference;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+@Getter
+@Slf4j
+public class ColorDiffFilter implements Filter {
+
+	private static final Pattern RGB = Pattern.compile( "rgb\\((\\d+),\\s?(\\d+),\\s?(\\d+)(,\\s?(\\d+))?\\)" );
+	private static final double MAX_DISTANCE = 255;
```

Comment:
`double` declared, but `int` given.

Reply:
Why is that a problem? We need this to be a double later in the calculation.

Reply:
No need for a conversion here. You can just initialize it with a double:
`private static final double MAX_DISTANCE = 255.0;`